exporter/datadogexporter/examples/k8s-chart/daemonset.yaml

```diff
@@ -50,9 +42,6 @@ spec:
             - name: varlogpods
               mountPath: /var/log/pods
               readOnly: true
-            - name: varlibdockercontainers
```

Comment:
Why don't we need this anymore?

Reply:
I don't know why we needed it to begin with to be honest. In all my tests I haven't encountered a need for it.

Reply:
It was added in #14795, maybe it's used for logs? @dineshg13 can you have a look?